v1/src/main/java/com/google/cloud/teleport/spanner/common/SizedType.java

```diff
@@ -99,10 +108,32 @@ public static String typeString(Type type, Integer size) {
     throw new IllegalArgumentException("Unknown type " + type);
   }
 
+  public static String typeString(Type type, Integer size, Integer arrayLength) {
```

Comment:
Why `Integer` and not `int` for the `arrayLength`? I don't see any places where the argument can be `null`.

Reply:
Agreed.